src/pushsource/type_aliases.pyi

```diff
@@ -0,0 +1,17 @@
+import datetime
```

Comment:
Can we please move this file into _impl to indicate it's a private implementation detail?

We don't want .pyi files in any other projects doing "from pushsource.type_aliases import ...".